keyboards/planck/keymaps/samuel/keymap.c

```diff
@@ -46,46 +42,38 @@ _COMMAND
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 
 [_DVORAK] =
-
 {{ KC_T_LRSE, KC_QUOT, KC_COMM, KC_DOT, KC_P, KC_Y, KC_F, KC_G, KC_C, KC_R, KC_L, KC_T_RRSE},
```

Comment:
Could you please use layout macros here instead of the raw matrix arrays? I think the one you want is `LAYOUT_planck_grid`. See the default keymap for an example.

Reply:
/me cries in [#7172](https://github.com/qmk/qmk_firmware/pull/7172/files#diff-8150f273cd8ed20384758e5ab878bc10)

Reply:
The keymap is auto generated. I will edit the python that generates it to make it up to new standards. Thanks for the suggestion.